widgetastic_manageiq.py

```diff
@@ -1589,3 +1589,19 @@ class chart(TimelinesChart):  # NOQA
     @property
     def is_displayed(self):
         return self.title.text == 'Timelines'
+
```

Comment:
My :+1:  I like the nested view , easy to understand which form belongs to which view and ofcourse no need to define buttons